doc/tracing.md

```diff
@@ -211,6 +211,58 @@ Arguments passed:
 4. The expected transaction fee as an `int64`
 5. The position of the change output as an `int32`
 
+### Context `mempool`
+
+#### Tracepoint `mempool:added`
+
+Is called when a transaction is added to the node's mempool. Passes information
+about the transaction.
+
+Arguments passed:
+1. Transaction ID (hash) as `pointer to unsigned chars` (i.e. 32 bytes in little-endian)
+2. Transaction virtual size as `uint64`
+3. Transaction fee as `int64`
+
+#### Tracepoint `mempool:removed`
+
+Is called when a transaction is removed from the node's mempool. Passes information
+about the transaction.
+
+Arguments passed:
+1. Transaction ID (hash) as `pointer to unsigned chars` (i.e. 32 bytes in little-endian)
+2. Removal reason as `pointer to C-style String` (max. length 9 characters)
+3. Transaction virtual size as `uint64`
+4. Transaction fee as `int64`
+
```

Comment:
probably makes sense to also pass the time the transaction was initially added to the mempool here. `CTxMempoolEntry` has a `GetTime()` method.

Reply:
It would probably make sense to include the time for all of the tracepoints added in this PR?

Reply:
Not necessarily. Might be redundant. When a transaction is added and you pass the tracepoint arguments _directly_ to a tracing script (can't think of a reason why you would want to store it in a BPF map first for a while), then the time is very close to `now()`. Similar for rejected transactions and for the replacement transaction during RBF (the time when the replaced transaction entered is passed).

Reply:
In reply to the original comment: oversight on my part; will push a fix.

In reply to adding event timestamps for all tracepoints: in omitting the actual event timestamps as tracepoint parameters, I followed the precedent set by other tracepoints, where, as laid out by @0xB10C, event timestamps are generated inside tracing scripts.